crates/contract/src/lib.rs

```diff
@@ -1117,6 +1121,30 @@ impl MpcContract {
         }
     }
 
+    /// Private endpoint to clean update votes from non-participants after resharing.
+    /// This can only be called by the contract itself via a promise.
+    #[private]
+    #[handle_result]
+    pub fn clean_non_participant_update_votes(&mut self) -> Result<(), Error> {
```

Comment:
Clean can mean a lot of things. We also have `clear_votes_from_non_participants`. Let's make it more explicit and consistent
```suggestion
    pub fn remove_non_participant_update_votes(&mut self) -> Result<(), Error> {
```